README.md

```diff
@@ -10,6 +10,9 @@ Please check out common DSC Resources [contributing guidelines](https://github.c
 
 ## Resources
 
+# AutoUpdates
+Configures Automatic Updates on a machine using the registry keys under HKLM:\Software\Policies\Microsoft\Windows\WindowsUpdate.
+
```

Comment:
Could you please add section
### Unreleased 
with description of your changes on line 26? (under ## Versions).

We recently added this requirement to our contribution guide: https://github.com/PowerShell/DscResources/blob/master/CONTRIBUTING.md#lifecycle-of-a-pull-request